### PR TITLE
Clear arrows from player's body on death (FIXED)

### PR DESCRIPTION
**Pull request by FabrisNick**

### CODE REVIEW

**Comment by andrei1058** [comment]
Did you even import CraftPlayer? I don't think this will work with all version...

**Comment by ajgeiss0702** [comment]
It looks like CraftPlayer was already imported from other things. It compiles (as you can see in the checks).

I would also like to see that it works on all versions.

**Comment by andrei1058** [comment]
can someone test this on 1.12 and 1.13 or newer?

**Comment by FabrisNick** [comment]
Tested with 1.8.8, 1.9.4, 1.12.2, 1.15.2, 1.16.5

File: versionsupport_1_16_R1/src/main/java/com/andrei1058/bedwars/support/version/v1_16_R1/v1_16_R1.java [hunk 1]
```diff
@@ -670,6 +670,7 @@ public void playRedStoneDot(Player player) {
 
     @Override
     public void clearArrowsFromPlayerBody(Player player) {
-        ((CraftPlayer)player).getHandle().getDataWatcher().set(new DataWatcherObject<>(10, DataWatcherRegistry.b),-1);
+        System.out.println("1.16_R1");
```
Comment (andrei1058) [inline versionsupport_1_16_R1/src/main/java/com/andrei1058/bedwars/support/version/v1_16_R1/v1_16_R1.java]: Remove this

**Review by andrei1058 — CHANGES_REQUESTED** [review]
please align with master. 1.18 support has been added

File: versionsupport_1_16_R1/src/main/java/com/andrei1058/bedwars/support/version/v1_16_R1/v1_16_R1.java (hunk 1 above)
Comment (FabrisNick) [reply versionsupport_1_16_R1/src/main/java/com/andrei1058/bedwars/support/version/v1_16_R1/v1_16_R1.java]: Done

**Review by andrei1058 — APPROVED** [review]

**Pull request merged**